tests/queries/0_stateless/02113_base64encode_trailing_bytes.sql

```diff
@@ -0,0 +1,11 @@
+set log_queries=1;
+DROP TABLE system.query_log;
```

Comment:
Please don't do this, this affects the whole cluster (and all tests).

You can use things like `$CLICKHOUSE_DATABASE` and so on to create distinct queries.